src/geometamaker/geometamaker.py

```diff
@@ -261,7 +261,7 @@ def describe_raster(source_dataset_path, scheme):
         b = i + 1
         bands.append(models.BandSchema(
             index=b,
-            gdal_type=info['datatype'],
+            gdal_type=gdal.GetDataTypeName(info['datatype']),
```

Comment:
Not related to the rest of the PR, but we should have been using the human-readable names all along.